modules/endpoint-group/main.tf

```
@@ -1,5 +1,20 @@
 locals {
-  enabled = module.this.enabled
+  enabled                 = module.this.enabled
+  endpoint_configurations = try(var.config.endpoint_configuration, null) != null ? var.config.endpoint_configuration : []
```

@aknysh This is what you said, but not what you meant. We want to handle the case where `var.config.endpoint_configuration = ""` properly, so I suggest this:
```suggestion
  endpoint_configurations = try(length(var.config.endpoint_configuration), 0) > 0 ? var.config.endpoint_configuration : []
```

---

Resolved

---

thanks @Nuru 
I just mentioned the idea, not the final solution